.gitignore

```diff
@@ -39,3 +39,6 @@ tests/end2end/tests/*
 tests/lightgbm/tests/*
 tests/build/*
 tests/tests/
```

Comment:
should rename tests to some unusual name like _dump to avoid name misleading?

Reply:
Maybe, I would remove end2end testing as it mostly test keras. In a next PR...

Reply:
Probably not, since end2end also includes tests for CoreML.